assets/script.js

```diff
@@ -0,0 +1,203 @@
+// Declaring global variables
+var today = dayjs();
+var full_name = "";
+var index;
+var tarotDesc = [];
+var tarotName = [];
+var historySign;
+var historyName;
+var tarotID;
+
+today = today.format('MM/DD/YYYY');
+// Empty the current horoscope section of the page
+function clear() {
+    console.log("inside clear");
+    $(".curr-hor").empty();
+}
+// function to display tarto information with a modal for the selected card
+function updatetarot(tarotData) {
+    console.log("inside     update tarot", tarotData);
+    for (i = 0; i < tarotData.cards.length; i++) {
+        console.log("For card number: ", i, " info: ", tarotData.cards[i]);
+        console.log("You drew a ", tarotData.cards[i].type, " card from the ", tarotData.cards[i].suit, " suit ");
+        console.log("Desc:  ", tarotData.cards[i].desc);
+        tarotName[i] = tarotData.cards[i].name;
+        tarotDesc[i] = tarotData.cards[i].desc;
+    }
+}
+// function for AHAX request to get random tarot cards
+function tarot() {
+    console.log("Inside tarot function");
+    $.ajax({
+        type: 'GET',
+        url: 'https://rws-cards-api.herokuapp.com/api/v1/cards/random?n=16',
+        success: function (tarotData) {
+            console.log(tarotData);
+            updatetarot(tarotData);
+        }
+    });
+}
+
+// function to update the page with horotscope information 
+function updatePage(horData, sign) {
+
+
+    sign = sign.toUpperCase();
+
+
+    var $currentHorEl = $("<ul>");
+    $currentHorEl.addClass("curr-hor");
+    $("#horoscope").append($currentHorEl);
+
+    console.log(horData);
+    var color = horData.color;
+    var $currentHorItem = $("<li>");
+    $(".curr-hor").append($currentHorItem);
+    $currentHorItem.append("<h6> Hello " + full_name + "!!. This is your Horoscope for " + today + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your sun sign is " + sign + " based on your birthday between " + horData.date_range + "</h6>" + "<br>");
+    $currentHorItem.append("<h6>" + horData.description + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Mood: " + horData.mood + " </h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky color is: " + horData.color + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky number is: " + horData.lucky_number + "</h6>" + "<br>");
+    $currentHorItem.append("<h6> Your lucky time is: " + horData.lucky_time + "</h6>" + "<br>");
+    console.log("Color is: ", color);
+    if (window.document.title !== "Astrozoders") {
```

Comment:
You should probably have separated out some of this page specific code in to javascript files that would only be included on the relevant pages.

It's not a good idea to co-mingle code across various pages.

Also, you want to load as little code as possible on the client side to improve performance